bump_version.sh

```diff
@@ -0,0 +1,51 @@
+#!/usr/bin/env bash
+
+# bump_version.sh (show|major|minor|patch|prerelease|build)
+
+set -o nounset
+set -o errexit
+set -o pipefail
+
+VERSION_FILE=src/example/_version.py
```

Comment:
This should be:  `VERSION_FILE=aws_jobs/_version.py`

Reply:
whoops, nice catch. Fixed in daae9d4